src/ResourceManager/ServiceFabric/AzureRM.ServiceFabric.psd1

```diff
@@ -75,13 +75,13 @@ FunctionsToExport = @()
 CmdletsToExport = 'Add-AzureRmServiceFabricApplicationCertificate', 
                'Add-AzureRmServiceFabricClientCertificate', 
                'Add-AzureRmServiceFabricClusterCertificate', 
-               'Add-AzureRmServiceFabricNode', 'Add-AzureRmServiceFabricNodeType', 
+               'Add-AzureRmServiceFabricNode', 
+               'Add-AzureRmServiceFabricNodeType', 
                'Get-AzureRmServiceFabricCluster', 
                'New-AzureRmServiceFabricCluster', 
                'Remove-AzureRmServiceFabricClientCertificate', 
                'Remove-AzureRmServiceFabricClusterCertificate', 
                'Remove-AzureRmServiceFabricNode', 
-               'Remove-AzureRmServiceFabricNodeType', 
```

Comment:
@QingChenmsft ok this will not be possible. Any reasons you want to remove this cmdlet?
Please add detailed explanation in description as to why are you removing this cmdlet?
Also someone can still import binary module (.dll) and can still use your cmdlet.